src/main/java/hudson/MockJenkins.java

```diff
@@ -1,63 +1,96 @@
-package hudson;
-
-import hudson.init.InitMilestone;
-import hudson.model.Hudson; //only needed until Hudson reference is removed from the ExtensionList.
-import jenkins.model.Jenkins;
-import org.jenkinsci.infra.tools.HyperLocalPluginManager;
-
-import static org.mockito.Mockito.*;
-import org.mockito.invocation.InvocationOnMock;
-import org.mockito.stubbing.Answer;
-
 /**
  * Mock for the Jenkins/Hudson object required for processing
  * the various steps.  This became very important after introducing
  * mocking as pretty much every class wants to know what's happening
  * within Jenkins.  Nothing here actually starts the process, and
  * provides a very bland "Jenkins" startup
  *
- * This mock technically should be for Jenkins.class.  However,
- * ExtensionList pretty much requires that the object also be of
- * type Hudson.  Since Mockito works by creating a subclass of the
- * desired class, it has to be of Hudson.  This should be changed
+ * This mock technically should be for Jenkins.class.  However, 
+ * ExtensionList pretty much requires that the object also be of 
+ * type Hudson.  Since Mockito works by creating a subclass of the 
+ * desired class, it has to be of Hudson.  This should be changed 
```

Comment:
Optional.  Prefer to not have trailing white space in the file.
```suggestion
 * This mock technically should be for Jenkins.class.  However,
 * ExtensionList pretty much requires that the object also be of
 * type Hudson.  Since Mockito works by creating a subclass of the
 * desired class, it has to be of Hudson.  This should be changed
```